src/scry/protocol/cancel_params.cr

```diff
@@ -1,9 +1,11 @@
 require "json"
 
 module Scry
-  struct CancelParams
-    JSON.mapping({
-      id: Int32 | String,
-    }, true)
+  module Protocol
```

Comment:
you can do blend the 2 modules to `module Scry::Protocol`, and avoid adding another indentation inside

Reply:
I guess thats personal preference.  I always prefer to keep module declarations separate, maybe its from my Ruby background where it was always advised to do it this way to avoid the situation where a module wasn't defined prior to writing the Path.  I think this way is alway more clear that `Protocol` is a module.